.travis.yml

```diff
@@ -12,30 +12,22 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 5.6
+    - php: 7.3
       env:
         - DEPS=latest
-    - php: 7
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.1
+    - php: nightly
```

Comment:
Could you change the travis configuration as in laminas/laminas-servicemanager#64?
Travis added PHP 8.0 [yesterday](https://travis-ci.community/t/php-8-0-missing/10132/23) support for the matrix and thus we can add `8.0` aswell as `nightly` where nightly is allowed to fail.

There is an example in the linked PR 👍

Reply:
Updated. I wasn't quite sure about adding the env vars here, but I'm happy to do that if that's desired: https://github.com/laminas/laminas-servicemanager/blob/cd5686e97ca9637ee69244a7b2396bbb5113ec4e/.travis.yml#L19-L22

Also, any thoughts about including cs-check/cs-fix changes in this request too?

Reply:
@stringerbell
> Also, any thoughts about including cs-check/cs-fix changes in this request too?

Please create a separate pull request for this because the topic of this pull request and the related issue report is the support of PHP 8 and this is enough. It is clearer for the reviewers and makes it easier for you too.

Thanks in advance! 👍